modules/bgsegm/src/bgfg_lsbp.cpp

```diff
@@ -0,0 +1,228 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
+//
+//  IMPORTANT: READ BEFORE DOWNLOADING, COPYING, INSTALLING OR USING.
+//
+//  By downloading, copying, installing or using the software you agree to this license.
+//  If you do not agree to this license, do not download, install,
+//  copy or use the software.
+//
+//
+//                          License Agreement
+//                       (3-clause BSD License)
+//                     For BackgroundSubtractorCNT
+//               (Background Subtraction based on Counting)
+//
+// Copyright (C) 2016, Sagi Zeevi (www.theimpossiblecode.com), all rights reserved.
```

Comment:
@StevenPuttemans  The code is mine. I blindly took license header from the nearest file (bgfg_subcnt.cpp) - that was a mistake :)